airflow/providers/google/cloud/operators/vertex_ai/custom_job.py

```diff
@@ -1111,10 +1129,19 @@ def __init__(
         self,
         *,
         script_path: str,
+        region: str,
         requirements: Sequence[str] | None = None,
+        dataset_id: str | None = None,
+        impersonation_chain: str | Sequence[str] | None = None,
         **kwargs,
     ) -> None:
-        super().__init__(**kwargs)
+        # FIXME: allow assignments in the super() constructor
```

Comment:
What is this note about?

Reply:
I need to figure out how to deal with attributes that are initialized at the parent's level -
Let's say that the parent has a mandatory template field `foo`, it will be init. by `super().__init__(foo=foo)`. As of the current implementation, I also need to set it in the class itself to detect it in the pre-commit.